WebColock.Tests/ClockControllerTests.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using Microsoft.VisualBasic;
```

Comment:
Do czego potrzebujecie VB? ;)

Reply:
W sumie to nie wiem.. :)
Automatycznie VS to dodał. Już wyrzucone.